README.md

```diff
@@ -42,8 +42,11 @@ represent the tiles.
 
 ## A1.1
 
-«replace this with your answer»
-
+S -> C
+S -> C S C
+C -> r
+C -> g
+C -> b
 
```

Comment:
# 0

The rule "S -> C S C" means "expand S to: choose a C, choose an S, and choose another C". So there's no guarantee that S will start and end the same.

I was looking for something like

       S -> r | g | b
       S -> r  S  r
       S -> g  S  g
       S -> b  S  b